### DISCUSSION TITLE
Creating ticket with Email Scanner

### DISCUSSION THREAD

**Discussion by jlatka** [YetiForceCompany/YetiForceCRM#17140] [How to]
Hello, 

I used your [video](https://www.youtube.com/watch?v=0JBckYE9ZuI) to set up creating a ticket using Email Scanner. It works fine, but... is it possible to set anywhere an option to have a ticket created for every message that comes into the mailbox?

Currently it looks like I have to read the message in the YetiForce mailbox for the ticket to create. I would like a ticket to be created for every email that comes in, whether it is read or not. 

If this is not possible, please help me on how I can do what I described above. 

The second question is about the customer support portal - how can customers create tickets through the WEB site, so that it also appears as a ticket?

I would be very grateful for help

**Comment by jlatka** [comment]
BUMP

Anyone any ideas? It's really confusing, I didn't resolved it for a week :(

**Comment by jlatka** [comment]
BUMP

Missclicked, it's not closed

**Comment by rskrzypczak** [comment]
> Currently it looks like I have to read the message in the YetiForce mailbox for the ticket to create. 
I would like a ticket to be created for every email that comes in, whether it is read or not.

not at all.
just set the mail scanner actions and it will scan your emails in the background whether you enter your mail client or not.
You can see the mail scanner configurations in the video you attached in your message. However, for the mail scanner to work properly, cron must be running.

> The second question is about the customer support portal - how can customers create tickets through the WEB site, so that it also appears as a ticket?

use Api https://doc.yetiforce.com/api/
